massa-node/base_config/config.toml

```diff
@@ -14,7 +14,7 @@
     # max number of arguments per RPC call
     max_arguments = 128
     # path to the openrpc specification file used in `rpc.discover` method
-    openrpc_spec_path = "base_config/openrpc.json"
+    openrpc_spec_file = "base_config/openrpc.json"
```

Comment:
Not very important but here we specify a filepath so I would rename it to filepath to be really explicit?

Reply:
I don't think "file" is better than "path" because it's a "path". "path" and "filepath" are both ok but as we are using "path" everywhere (and a lot of projects does) I don't see any reason to rename "filepath" everywhere

Reply:
In massa `_path` wins compared to `_file`